nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -85,9 +85,9 @@ extide/gradle/external/gradle-7.4-bin.zip platform/libs.testng/external/jcommand
 extide/gradle/external/gradle-7.4-bin.zip platform/o.apache.commons.io/external/commons-io-2.6.jar
 extide/gradle/external/gradle-7.4-bin.zip enterprise/cloud.oracle/external/jsr305-3.0.2.jar
 extide/gradle/external/gradle-7.4-bin.zip platform/o.apache.commons.codec/external/commons-codec-1.15.jar
-extide/gradle/external/gradle-7.4-bin.zip platform/libs.asm/external/asm-9.2.jar
-extide/gradle/external/gradle-7.4-bin.zip platform/libs.asm/external/asm-commons-9.2.jar
-extide/gradle/external/gradle-7.4-bin.zip platform/libs.asm/external/asm-tree-9.2.jar
+extide/gradle/external/gradle-7.4-bin.zip platform/libs.asm/external/asm-9.3.jar
```

Comment:
Well. At the moment Gradle 7.4 comes with asm 0.9.2 so technically these lines could be removed as there would be no clash. On the other hand, it would be fine when newer Gradle would switch to 0.9.3, so I'd prefer to have it this way.

This is just an informational comment.